packages/repository-tests/src/types.repository-tests.ts

```diff
@@ -9,6 +9,7 @@ import {
   juggler,
   Options,
 } from '@loopback/repository';
+import {TransactionRepository} from '@loopback/repository/src';
```

Comment:
This import from `src` seems to be problematic.

Reply:
+1

```suggestion
import {TransactionRepository} from '@loopback/repository';
```